### DISCUSSION TITLE
Mathematical physics

### DISCUSSION THREAD

**Discussion by Alexander7754** [ReScience/ReScience#113] [Ideas]
If energy is removed from physics, then pure mathematics remains. If the values of physical constants are replaced by structures of natural numbers, then a static model of numerical complexes arises, which are in certain periodic orders. If the compositions of certain numerical orders are combined in a certain sequence, then a functional connection will arise between them - a connection that determines energy. If there is an argument C between argument A and argument B, then this is a material object.

I propose a number of structural matrices that solve "cold" fusion according to Mössbauer nuclei.

**Comment by khinsen** [comment]
I have blocked the author of this discussion because this is obviously spam.
